pyserini/fusion/_base.py

```diff
@@ -84,3 +85,28 @@ def interpolation(runs: List[TrecRun], alpha: int = 0.5, depth: int = None, k: i
     scaled_runs.append(runs[1].clone().rescore(method=RescoreMethod.SCALE, scale=(1-alpha)))
 
     return TrecRun.merge(scaled_runs, AggregationMethod.SUM, depth=depth, k=k)
+
+
+def average(runs: List[TrecRun], depth: int = None, k: int = None):
+    """Perform fusion by averaging on a list of ``TrecRun`` objects.
+    new_score = sum(scores) / num_runs
```

Comment:
I don't think you need to explain how to compute average... :)

Reply:
Hahaha. Will remove.